src/service-override/views.ts

```diff
@@ -420,5 +420,20 @@ export {
   OpenEditor,
   IEditorOptions,
   IResolvedTextEditorModel,
-  IReference
+  IReference,
+
+  IViewDescriptorService,
+  IActivityService,
+  IHoverService,
+  IExplorerService,
+  IStatusbarService,
```

Comment:
The convention for the moment is to export service identifiers from services.ts

Reply:
Alright, changed it so everythings exported from the services file